plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -524,6 +534,25 @@ struct Database {
         stmt.execute;
     }
 
+    /** Set mutantStatus by using MutantId.
+	Can be used both for ignoring and "accepting" mutants.
+     */
+    void markMutant(ulong id, Mutation.Status to_status, string rationale = "") @trusted {
+        Mutation.Status from_status = Mutation.Status(getMutationStatus(MutationId(id)));
+
+        // mark mutantStatus in mutationStatus-table
+        const s = format!"UPDATE %s SET status=%s WHERE id IN (SELECT st_id FROM %s WHERE id = %s)"(
+                mutationStatusTable, to_status.to!long, mutationTable, id.to!long);
+        auto stmt = db.prepare(s);
+        stmt.execute;
+
+        // add mutant to separate table for marked mutants
+        db.run(buildSchema!MarkedMutant); // TODO: can probably be done somewhere else
+        db.run(insert!MarkedMutant.insert, MarkedMutant(MutationId(id),
+                Path(getPath(MutationId(id))), from_status, to_status,
```

Comment:
fetching the path should be done before this call because it can fail if the user provides an invalid ID